src/MultiOwnable.sol

```diff
@@ -52,8 +54,9 @@ contract MultiOwnable {
     /// @notice Thrown when `owner` argument does not match owner found at index.
     ///
     /// @param index The index of the owner to be removed.
-    /// @param owner Owner to be removed which did not match owner found at index.
-    error WrongOwnerAtIndex(uint256 index, bytes owner);
+    /// @param expectedOwner The owner passed in the remove call.
+    /// @param actualOwner The actual owner at `index`.
+    error WrongOwnerAtIndex(uint256 index, bytes expectedOwner, bytes actualOwner);
```

Comment:
@stevieraykatz this felt nice to have to me

Reply:
Agreed -- could help track down malicious replays